File: sdk/approute-sdk/golang/enum/balance_category.go
package enum

// BalanceCategory represents a balance-transaction category.
type BalanceCategory string

const (
	BalanceFunding     BalanceCategory = "funding"
	BalanceRefund      BalanceCategory = "refund"
	BalanceWithdraw    BalanceCategory = "withdraw"
	BalanceShop        BalanceCategory = "shop"
	BalanceDirectTopUp BalanceCategory = "direct-top-up"
)

File: sdk/approute-sdk/javascript/src/enums/balance-category.ts
/**
 * Balance transaction categories.
 */
export const BalanceCategory = {
  FUNDING: "funding",
  REFUND: "refund",
  WITHDRAW: "withdraw",
  SHOP: "shop",
  DIRECT_TOP_UP: "direct-top-up",
} as const;

export type BalanceCategory =
  (typeof BalanceCategory)[keyof typeof BalanceCategory];
